### DISCUSSION TITLE
documentation for updating dokploy

### DISCUSSION THREAD

**Discussion by Manubi** [Dokploy/dokploy#70] [General]
Hi,
is there any documentation how I can update dokploy to the latest version?

**Comment by Siumauricio** [comment]
Updating Procedure:

Navigate to Settings > Server.
Upon entry, the system will automatically begin downloading the latest server image. Please wait approximately 2 minutes for the download to complete.
Once the download is finished, an "Update Server" button will appear. Click this button to proceed with the server update.

if you can't see the button simply reload the page


![Screenshot 2024-05-10 at 8 49 10 AM](https://github.com/Dokploy/dokploy/assets/47042324/40e94f2f-d519-467c-b194-160af30cd839)
